src/app/model/sell/sell.ts

```diff
@@ -0,0 +1,19 @@
+export interface Sell {
+    customerInformation: CustomerInformation,
+    totalAmount: number
+    cacke: CackeSold;
+}
+
+interface CustomerInformation {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.